docs/dictionary.rst

```diff
@@ -0,0 +1,1101 @@
+
+.. note:: This term as we define it here is associated with Dockstore and may have different definitions in other contexts.  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict faceted search:
+
+faceted search
+--------------
+	A type of search which allows users to narrow down their results based upon certain aspects of the things being searched. On Dockstore, our faceted search at <https://dockstore.org/search> allows users to narrow down their search to a particular workflow language, author, or other fields.  
+
+Further reading: `<https://en.wikipedia.org/wiki/Faceted_search>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict FAIR:
+
+FAIR
+----
+pronounced "fair", rhymes with pear  
+
+*abbreviation for* Findable, Accessible, Interoperable, and Reusable  
+
+	A set of guidelines to improve the Findability, Accessibility, Interoperability, and Reuse of digital assets. This concept is often applied to data, but can be applied to other assets such as workflows.  
+
+Further reading: `<https://www.go-fair.org/fair-principles/>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict GA4GH:
+
+GA4GH
+-----
+*abbreviation for* Global Alliance For Genomics and Health  
+
+	A network of public and private institutions which aims to accelerate progress in genomic research and human health by cultivating a common framework of standards and harmonized approaches for effective and responsible genomic and health-related data sharing.  
+
+Further reading: `<https://www.ga4gh.org/>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict Galaxy:
+
+Galaxy
+------
+	An open-source platform that uses :ref:`dict FAIR` principles, most well-known for its web-based UI which can be used to run a variety of bioinformatics tools.  
+
+Further reading: `<https://galaxyproject.org/>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict Galaxy workflow:
+
+Galaxy workflow
+---------------
+	A type of :ref:`dict workflow` that follows the standards of the :ref:`dict Galaxy` execution system. Dockstore supports the registration of Galaxy workflows with the file extension .ga  
+
+Further reading: `<https://galaxyproject.org/learn/advanced-workflow/>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict GCP:
+
+GCP
+---
+*abbreviation for* Google Cloud Platform  
+
+	A backend used for cloud computing and cloud storage hosted by Google. :ref:`dict Terra` is an example of a system that runs on a GCP backend. When running workflows on these backends, make sure to account for the storage needed for your workflow, as GCP compute backends do not automatically scale their storage size at runtime. GCP backends allow you to make use of Google's :ref:`dict preemptible` feature, which may reduce the cost of running workflows.  
```

Comment:
Same deal as with AWS, pick a more well known example than Terra and then lead to it